sdk/inc/azure/core/az_context.h

```diff
@@ -52,24 +52,24 @@ struct az_context
 extern az_context az_context_app;
 
 /**
- * @brief az_context_with_expiration creates a newexpiring az_context node that is a child of the
- * specified parent.
+ * @brief az_context_create_with_expiration creates a newexpiring az_context node that is a child of
+ * the specified parent.
  *
  * @param[in] parent The az_context node that the new node is to be a child of; passing NULL sets
  * the parent to az_context_app.
  * @param[in] expiration The time when this new child node should be canceled
  * @return The new child az_context node
  */
 AZ_NODISCARD AZ_INLINE az_context
-az_context_with_expiration(az_context const* parent, int64_t expiration)
+az_context_create_with_expiration(az_context const* parent, int64_t expiration)
 {
   return (az_context){ ._internal = { .parent = ((parent != NULL) ? parent : &az_context_app),
                                       .expiration = expiration } };
 }
 
 /**
- * @brief az_context_with_value creates a new key/value az_context node that is a child of the
- * specified parent.
+ * @brief az_context_create_with_value creates a new key/value az_context node that is a child of
```

Comment:
Since you are already modifying parts of the docs with this API and file, can you also just remove the redundant method name in the comments (limit it to just the code you are touching)? They aren't necessary.

See https://github.com/Azure/azure-sdk-for-c/issues/589

Reply:
TBH, having obsessive idealist tendencies, I would much rather just run through all comments and do it all at once, including `@retval`s `#AZ_OK`s, `[in]/[out]`s etc. But I'll do this one file, or otherwise the PR is going to be too big and out of scope.

Reply:
Makes sense.